File: evidence/generated-systems/angular/governance-on-angular/src/app/models/LegalHold.js

// Define collection and schema for LegalHold
export interface LegalHold {
    name:
	type : string
    reason:
	type : string
    issuedDate:
	type : Date
    releaseDate:
	type : Date
    Repositories:
 	type : [{ type: Schema.Types.ObjectId, ref: 'RecordsRepository' }]
    Records:
 	type : [{ type: Schema.Types.ObjectId, ref: 'Record_' }]
    Matter:
	type : Schema.Types.ObjectId
    HoldStatus:
 	type : String
#
    collection: 'legalHolds'
}

File: evidence/generated-systems/rubyonrails/governance-on-rails/governanceonrails/app/models/legalhold.rb
class LegalHold < ApplicationRecord
  enum HoldStatus: [:Active, :Released, :Superseded]


  has_many :Repositories, class_name: 'RecordsRepository'
  has_many :Records, class_name: 'Record_'
  has_many :Matter, class_name: 'Matter'

end
